Without this patch, the 'make install' attempts to 'install' a directory, which causes an error.

diff --git a/c++/src/build-system/Makefile.in.top b/c++/src/build-system/Makefile.in.top
index 1abe44f..d6000b3 100644
--- a/c++/src/build-system/Makefile.in.top
+++ b/c++/src/build-system/Makefile.in.top
@@ -51,7 +51,7 @@ install-toolkit:
 	    done
 	cd $(includedir0) && find * -name CVS -prune -o -print |\
             cpio -pd $(pincludedir)
-	$(INSTALL) -m 644 $(incdir)/* $(pincludedir)
+	cp -R $(incdir)/* $(pincludedir)
 ## set up appropriate build and status directories somewhere under $(libdir)?
 
 install-gbench:
